### ISSUE TITLE
HED训练中运行solve.py出现的问题

### ISSUE THREAD

**Issue by huangjiegis** [s9xie/hed#66]
I1105 20:11:10.021775 26811 layer_factory.hpp:77] Creating layer data

F1105 20:11:10.021813 26811 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: ImageLabelmapData
(known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Clip, Concat, ContrastiveLoss, Convolution, Crop, Data,
Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col,
ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter,
Pooling, Power, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax,
SoftmaxWithLoss, Split, Swish, TanH, Threshold, Tile, WindowData)
*** Check failure stack trace: ***
已放弃 (核心已转储)

老师您好，请问一下，在HED训练时，出现了上面的问题，该如何解决（已经在HED下编译caffe通过）

**Comment by cqray1990** [comment]
@huangjiegis  did you solver i meet this problem too

**Comment by yongshuaihuang** [comment]
@huangjiegis @cqray1990 did you solve this problem？
